sr_edc_launch/sr_edc.launch

```diff
@@ -76,4 +76,7 @@
   <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher">
     <param name="publish_frequency" type="double" value="100.0" />
   </node>
+
+  <!-- adding tactile state publisher using the tf prefix as a namespace -->
+  <include file="$(find sr_tactile_sensor_controller)/sr_tactile_sensor.launch"/>
```

Comment:
This doesn't impact bimanual system since they don't use the same launch file (right @toliver ?)